opentech/apply/utils/pdfs.py

```diff
@@ -0,0 +1,274 @@
+import os
+import io
+
+from reportlab.lib import pagesizes
+from reportlab.lib.colors import Color, white
+from reportlab.lib.styles import ParagraphStyle as PS
+from reportlab.lib.utils import simpleSplit
+from reportlab.pdfbase import pdfmetrics
+from reportlab.pdfbase.ttfonts import TTFont
+from reportlab.platypus import (
+    KeepTogether,
+    ListFlowable,
+    ListItem,
+    Paragraph,
+    SimpleDocTemplate,
+    Spacer,
+    Table,
+    TableStyle,
+)
+
+
+from bs4 import BeautifulSoup, NavigableString
+
+styles = {
+    'Question': PS(fontName='MontserratBold', fontSize=14, name='Question', spaceAfter=0, spaceBefore=18, leading=21),
+    'Normal': PS(fontName='NotoSans', name='Normal'),
+    'Heading1': PS(fontName='NotoSansBold', fontSize=12, name='Heading1', spaceAfter=4, spaceBefore=12, leading=18),
+    'Heading2': PS(fontName='NotoSansBold', fontSize=10, name='Heading2', spaceAfter=4, spaceBefore=10, leading=15),
+    'Heading3': PS(fontName='NotoSansBold', fontSize=10, name='Heading3', spaceAfter=4, spaceBefore=10, leading=15),
+    'Heading4': PS(fontName='NotoSansBold', fontSize=10, name='Heading4', spaceAfter=4, spaceBefore=10, leading=15),
+    'Heading5': PS(fontName='NotoSansBold', fontSize=10, name='Heading5', spaceAfter=4, spaceBefore=10, leading=15),
+}
+
+font_location = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'media', 'fonts')
+
+
+def font(font_name):
+    return os.path.join(font_location, font_name)
+
+
+pdfmetrics.registerFont(TTFont('Montserrat', font('Montserrat-Regular.ttf')))
+pdfmetrics.registerFont(TTFont('MontserratBold', font('Montserrat-Bold.ttf')))
+pdfmetrics.registerFont(TTFont('MontserratItalic', font('Montserrat-Italic.ttf')))
+pdfmetrics.registerFont(TTFont('MontserratBoldItalic', font('Montserrat-BoldItalic.ttf')))
+pdfmetrics.registerFontFamily(
+    'Montserrat',
+    normal='Montserrat',
+    bold='MontserratBold',
+    italic='MontserratItalic',
+    boldItalic='MontserratBoldItalic'
+)
+
+pdfmetrics.registerFont(TTFont('NotoSans', font('NotoSans-Regular.ttf')))
+pdfmetrics.registerFont(TTFont('NotoSansBold', font('NotoSans-Bold.ttf')))
+pdfmetrics.registerFont(TTFont('NotoSansItalic', font('NotoSans-Italic.ttf')))
+pdfmetrics.registerFont(TTFont('NotoSansBoldItalic', font('NotoSans-BoldItalic.ttf')))
+pdfmetrics.registerFontFamily(
+    'NotoSans',
+    normal='NotoSans',
+    bold='NotoSansBold',
+    italic='NotoSansItalic',
+    boldItalic='NotoSansBoldItalic'
+)
+
+DARK_GREY = Color(0.0154, 0.0154, 0, 0.7451)
+
+PAGE_WIDTH, PAGE_HEIGHT = pagesizes.legal
```

Comment:
Would be good to have this as a setting.

Reply:
Yeah, it probably should be. That's a bit more work as currently i'm using the global values throughout the methods. Can we add it as a backlog ticket?